docs/component-guides/notebooks.md

```diff
@@ -1,95 +1,139 @@
-# Notebooks Component Guide for Kubeflow on AWS
++++
+title = "Notebooks"
```

Comment:
The first half of this page has duplicated information from this existing README: https://github.com/awslabs/kubeflow-manifests/blob/main/components/notebook-dockerfiles/README.md

Reply:
I think that's fine because that file is not in the docs folder or on the site.

Reply:
ok, sounds good.